sympy/solvers/solveset.py

```diff
@@ -1318,10 +1319,10 @@ def _invert_modular(modterm, rhs, n, symbol):
             try:
                 remainder_list = nthroot_mod(rhs, expo, m, all_roots=True)
                 if remainder_list == []:
```

Comment:
This is probably a quite bad way to check for this? `if not remainder_list:`

Reply:
either should be fine; preference for the `if not` form.